styles.css

```diff
@@ -0,0 +1,81 @@
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    margin-bottom: 30px;
+    padding: 0;
+    background-color: beige;
+    color: black;
+}
+main {
+    display: flex;
```

Comment:
noticed that this was here, which is great! however I don't see flex being used in the children of the main element. as a stretch goal, see how you can incorporate flex by modifying the layout of these child elements that are contained in the parent flex container :)